### DISCUSSION TITLE
LibSQL vs better-sqlite3

### DISCUSSION THREAD

**Discussion by cr101** [tursodatabase/libsql#1449] [Q&A] [answered]
Hi,

What's the difference between LibSQL and [better-sqlite3](https://github.com/WiseLibs/better-sqlite3)?

**Comment by haaawk** [comment, marked as answer]
Better-sqlite3 is a pure sqlite3 library that allows you to have an in process database backed by a file on disk.
Libsql allows you to have an in process database either backed by a file on disk or by a remote server.
In a sense, Libsql is a superset of what better-sqlite3 is.
